apt/private/package_resolution.bzl

```diff
@@ -31,6 +31,15 @@ def _parse_dep(raw):
 
         version = (version_and_const[:vconst_i], version_and_const[vconst_i + 1:])
 
+    # Depends: python3:any
+    # is equivalent to
+    # Depends: python3 [any]
+    colon_i = raw.find(":")
```

Comment:
Can you add some tests for this here: https://github.com/GoogleContainerTools/rules_distroless/blob/main/apt/tests/package_resolution_test.bzl